theia-turn/turn-sprotty/src/views/ucm/startpoint.node.tsx

```diff
@@ -2,18 +2,23 @@
 
 import { svg } from 'snabbdom-jsx';
 import {IView, RenderingContext} from "sprotty/lib";
-import {MAPStartNode} from "../../models";
+import {TURNNode} from "../../models";
 import { VNode } from "snabbdom/vnode";
 
 export class StartPointNodeView implements IView {
 
     static readonly TYPE = 'turnnode:startpoint';
 
-    render(node: MAPStartNode, context: RenderingContext): VNode {
-        const radius = node.getDimension() / 2;
+    render(node: TURNNode, context: RenderingContext): VNode {
+        // const xOffset = Math.max(node.bounds.width, 0);
+        const yOffset = Math.max(node.bounds.height / 2, 0);
         const vnode = <g class-ucm-node={true} >
             {context.renderChildren(node)}
-            <circle cx={radius} cy={radius} r={radius} />
+            <circle class-body={true} class-selected={node.selected}
+                    cx={node.getDimension() / 2}
+                    cy={yOffset}
+                    r={node.getDimension() / 2}
```

Comment:
Are these 2 values flipped? Shouldn't the circle be centered in the middle, and the yOffset be enforced via the radius.
`cy = {node.getDimensions() / 2}`
`r = {yOffset}`